test/e2e/framework/framework.go

```diff
@@ -71,7 +100,14 @@ func init() {
 	TestDirPath = filepath.Join(homeDir, TestDir)
 	os.Setenv("HOME", TestDirPath)
 	pwd, _ := os.Getwd()
-	os.Setenv("PATH", pwd+HackToolsBinDir+":"+pwd+TanzuBinDir+":"+os.Getenv("PATH"))
+	index := strings.LastIndex(pwd, TanzuCLI)
+	if index < 0 {
+		log.Error(fmt.Sprintf("The current directory path %s does not have %s directory, %s and %s not set in PATH", pwd, TanzuCLI, TanzuCLIHackToolsBin, TanzuCLIBin))
+	} else {
+		hackBin := pwd[:index] + TanzuCLIHackToolsBin
+		cliBin := pwd[:index] + TanzuCLIBin
+		os.Setenv("PATH", hackBin+":"+cliBin+":"+os.Getenv("PATH"))
+	}
```

Comment:
Configuring `PATH` variable as part of the test means, we are assuming how binaries are organized. What if I am running these tests in some other environment where I have already configured all required `PATH` variables?

Is it possible to remove this logic from here and configure the `PATH` variable as part of `Makefile` from where we are invoking `make e2e-cli-core` target?

Reply:
updated in makefile